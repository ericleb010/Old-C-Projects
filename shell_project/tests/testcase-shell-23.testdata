sleep 15 &
echo have
sleep 10 &
echo got
echo none

