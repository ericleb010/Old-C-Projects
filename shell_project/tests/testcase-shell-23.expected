have
Child (<0>) exited -- success (Status: 0)
got
Child (<0>) exited -- success (Status: 0)
none
Child (<0>) exited -- success (Status: 0)
ERROR: Could not run ¸4‡.

------------------------------------
Welcome to the Assignment Shell!
At any time, you can call "help" to read the documentation.

	>>> 	>>> 	>>> 	>>> 	>>> 	>>> 	>>> Child (<0>) exited -- failure (Status: 0)

------------------------------------
Welcome to the Assignment Shell!
At any time, you can call "help" to read the documentation.

	>>> 	>>> 	>>> 	>>> 	>>> 	>>> 	>>> 
